### ISSUE TITLE
Tour docs use variable-width font

### ISSUE THREAD

**Issue by samestep** [asoffer/Icarus#117]
Currently [the tour](https://asoffer.github.io/Icarus/tour) page on the docs site uses a variable-width font instead of a monospace one:

![Screenshot 2022-06-30 3 17 04 PM](https://user-images.githubusercontent.com/8246041/176759884-4c2b01d4-1ec5-49a8-91ac-c4a149028e5a.png)

**Comment by asoffer** [comment]
Very strange. I've checked on several machines and it's displaying correctly. Can you peek at the javascript console and tell me if there's anything errors (other than a missing favicon)?

**Comment by samestep** [comment]
You're right, I checked this on iOS and macOS this morning and those both show fixed-width font; seems like Chrome OS is the only problematic one. This is the only message I see in the JavaScript console:
```
Error with Permissions-Policy header: Origin trial controlled feature not enabled: 'interest-cohort'.
```
